src/Compilers/Test/Core/FX/MonoHelpers.cs

```diff
@@ -11,5 +11,6 @@ namespace Microsoft.CodeAnalysis.Test.Utilities
     public static class MonoHelpers
     {
         public static bool IsRunningOnMono() => Roslyn.Test.Utilities.ExecutionConditionUtil.IsMonoDesktop;
```

Comment:
Should we rename this to `IsRunningOnMonoDesktop`?

Reply:
Looks to be a good idea. However I would do it in a separate PR. 
Please let me know if you insist to be done as part of this PR itself.

Reply:
Separate PR or just filing an issue is fine, thanks.